app/views/new.erb

```diff
@@ -0,0 +1,52 @@
+<h1 class="heading">Create a New Robot!</h1>
+<div class="col-md-12">
+  <div class="alert alert-info" role="alert">
+    Random information has been generated below - feel free to replace any and all with your own!
+  </div>
+</div>
+<div class="row col-md-12">
+  <form action="/robots" method="post">
```

Comment:
this form is pretty similar to the `edit` template. this is a common place where you'll see people use partials. partials are a way to DRY up our view code - basically, we render a view template within a view template to reuse code. if you want to dive deeper into this sooner than later, please please let me know 👍  i do believe we will go over it though at some point in mod 2